intermediate_source/fx_conv_bn_fuser.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+(beta) Building a Conv/BN fuser in FX
+*******************************************************
+**Author**: `Horace He <https://github.com/chillee>`_
+
+In this tutorial, we are going to use FX to do the following:
+
+1) Find patterns of conv/batch norm in the data dependencies.
+2) Fold the batch norm statistics into the convolution weights.
+
+Note that this optimization only works for models in inference mode.
```

Comment:
nit: a little more detail here is probably worth it, eg. something like "only works for models in inference model (i.e. `model.eval()`).